oxidation/libparsec/crates/client_types/src/local_device_file.rs

```diff
@@ -101,6 +101,24 @@ impl DeviceFile {
     }
 }
 
+#[serde_as]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct LegacyDeviceFilePassword {
+    #[serde_as(as = "Bytes")]
+    pub salt: Vec<u8>,
+    #[serde_as(as = "Bytes")]
+    pub ciphertext: Vec<u8>,
+    pub human_handle: Option<HumanHandle>,
+    pub device_label: Option<DeviceLabel>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "lowercase")]
+#[serde(tag = "type")]
+pub enum LegacyDeviceFile {
+    Password(LegacyDeviceFilePassword),
+}
```

Comment:
Since we have only 1 `LegacyDeviceFileType` can't we forget the enum ?

Reply:
I made this enum because of the field `type` which is always password in case of legacy device file.

Reply:
Could you add a comment for that ?

Another way would be to add a field named `type` in `LegacyDeviceFilePassword` :thinking: 
But that way serde check if `type` is set to a correct value (i.e.: `type == 'password'`)

Reply:
Done.